src/main/java/org/lwes/emitter/MulticastEventEmitter.java

```diff
@@ -44,27 +44,15 @@
  * @author Anthony Molinaro
  * @since 0.0.1
  */
-public class MulticastEventEmitter extends AbstractEventEmitter {
+public class MulticastEventEmitter extends DatagramSocketEventEmitter<MulticastSocket> {
 
   private static transient Log log = LogFactory.getLog(MulticastEventEmitter.class);
 
-  /* the actual multicast socket being used */
-  private MulticastSocket socket = null;
-
-  /* the multicast address */
-  private InetAddress address = null;
-
-  /* the multicast port */
-  private int port = 9191;
-
   /* the multicast interface */
-  private InetAddress iface = null;
+  protected InetAddress iface = null;
```

Comment:
actually address in the parent class replaces iface, but i need to make a fix to use it everywhere in this class.